app/views/voter/show.html.erb

```diff
@@ -58,14 +58,174 @@
+                          <td style='border:none;'><%= @voter.phone_number_display || "No phone number given"%></td> <!-- need to hyperlink -->
+                        </tr>
+                        <tr>
+                          <th>Email</th>
+                          <td style='border:none;'><%= @voter.email || "No email given" %></td> <!-- need to hyperlink -->
+                        </tr>
+                        <tr>
+                          <th>Registration</th>
+                          <td style='border:none;'><%= @voter.voter_registration_status ? @voter.voter_registration_status_display : "Unavailable "%></tr>
+                        </tr>
+                        <tr>
+                          <th>Voted in 2020</th>
+                          <td style='border:none;'>Unavailable</td>
+                        </tr>
+                        <tr>
+                          <th>Vote Plan</th>
+                          <td style='border:none;'>Unavailable</td>
+                        </tr>
+                        <tr>
+                          <th>Contact Status</th>
+                          <td style='border:none;'><%= @voter.last_call_status_display %></td>
+                        </tr>
+                    </table>
+                  </div>
+                </div>
+              </div>
+          </div>
+
+          <div id="edit-voter-profile" style="padding-top: 10px;">
+            <div id="edit-voter-profile-info">
+              <div class="box contacts">
+
+                <div class="field">
+                  <label class="label">Email</label>
+                    <div class="control has-icons-left has-icons-right">
+                      <%= form_tag voter_path(@voter), method: :patch do %>
+                        <%= email_field_tag :email, @voter.email, class: "input"%>
+                        <!-- registration status is only cosmetic, does not work on back end as of now -->
+                        <%= select_tag :voter_registration_status, raw("<option>Registered in PA HD 82</option><option>Registered in PA out of HD 82</option><option>Registered out of state</option><option>Unregistered</option><option>Pending registration in PA HD 82</option>"), class:"select", style: "margin-top: 10px;"%>
+                        <%= text_area_tag :notes, @voter.notes, class:"textarea", placeholder:"Place notes here", style: "margin-top: 10px;" %>
+                        <%= submit_tag "Save changes", class:"button is-success is-medium is-light", style: "margin-top: 10px;" %>
+                      <% end %> 
+                      <span class="icon is-small is-left">
+                        <i class="fas fa-envelope"></i>
+                      </span>
+                    </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <!-- keeping this for showing off the UI in demos
+          <div id="survey-voter-data" style="padding-top: 10px;">
```

Comment:
also hardcoded without using ruby forms, going to change to a proper form soon. just keeping this here for some demo stuff